typhos/positioner.py

```diff
@@ -168,13 +171,35 @@ def _start_status_thread(self, status):
         thread.status_finished.connect(self._status_finished)
         thread.start()
 
+    def _get_timeout(self, value, settle_time):
+        """Use positioner's configuration to select a timeout."""
+        pos_sig = getattr(self.device, self._readback_attr, None)
+        vel_sig = getattr(self.device, self._velocity_attr, None)
+        acc_sig = getattr(self.device, self._acceleration_attr, None)
```

Comment:
Are those attributes using cached values at the signal or always doing a get?
Could this cause a problem since this is being calculated at every movement request?

Reply:
Looking at ophyd, these gets will not used any cached values. Perhaps there is a better way to get these since the screen should already be monitoring them in some way for the display.

Reply:
I think that they will be fine. Sorry my brain is damaged today. If it becomes a big deal we can always refactor later on.